packages/bpk-component-barchart/src/BpkBarchartBar.js

```diff
@@ -51,35 +51,53 @@ const BpkBarchartBar = (props) => {
     onClick,
     outlier,
     selected,
+    padding,
```

Comment:
I was looking for an explination of what this value is but I couldn't find one because the readme.md isn't updated